app/namespace.go

```diff
@@ -33,23 +54,30 @@ var (
 		Usage:   "The path to the ca pem file",
 		Aliases: []string{"f"},
 	}
-
 	caCertificateFingerprintFlag = &cli.StringFlag{
 		Name:    caCertificateFingerprintFlagName,
 		Usage:   "The fingerprint of to the ca certificate",
 		Aliases: []string{"fp"},
 	}
+	searchAttributeFlag = &cli.StringSliceFlag{
+		Name:     searchAttributeFlagName,
+		Usage:    fmt.Sprintf("Flag can be used multiple times; value must be \"name=type\"; valid types are: %v", getSearchAttributeTypes()),
+		Aliases:  []string{"sa"},
+		Required: true,
```

Comment:
why is this required?

Reply:
(or where is this getting used? maybe I have missed it)

Reply:
this is used by the update namespace command where the flag is required.